{"name": "test_controller_connectivity", "status": "broken", "statusDetails": {"message": "AttributeError: 'Controller' object has no attribute 'get_sdk_version'", "trace": "fixtures_ver = <fixtures_1x.Fixtures_1x object at 0x7fcf89f8bc10>\n\n    @pytest.fixture(scope=\"session\")\n    def get_sdk_version(fixtures_ver):\n>       version = fixtures_ver.get_sdk_version()\n\nconftest.py:276: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <fixtures_1x.Fixtures_1x object at 0x7fcf89f8bc10>\n\n    def get_sdk_version(self):\n>       version = self.controller_obj.get_sdk_version()\nE       AttributeError: 'Controller' object has no attribute 'get_sdk_version'\n\nfixtures_1x.py:63: AttributeError"}, "description": "Test case to verify cloud Controller Connectivity", "start": 1639441762192, "stop": 1639441762192, "uuid": "dd7253fa-698e-443e-8d04-e8af872c4dc5", "historyId": "29ddc2cc8da971f63eac960bf88db5ea", "testCaseId": "ddf09b31e4c32be1cdfceb0712bd475e", "fullName": "test_connectivity.TestResources#test_controller_connectivity", "labels": [{"name": "tag", "value": "sanity"}, {"name": "tag", "value": "test_resources"}, {"name": "tag", "value": "test_cloud_controller"}, {"name": "tag", "value": "sanity_55"}, {"name": "tag", "value": "uc_sanity"}, {"name": "suite", "value": "test_connectivity"}, {"name": "subSuite", "value": "TestResources"}, {"name": "host", "value": "testing-164-mngfr"}, {"name": "thread", "value": "8-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_connectivity"}], "links": [{"type": "test_case", "name": "test_controller_connectivity"}, {"type": "test_case", "name": "Test Resources"}]}